Branch-SDK/src/io/branch/referral/BranchConfig.java

```diff
@@ -0,0 +1,8 @@
+package io.branch.referral;
+
+/*
+ *
+ */
+public class BranchConfig {
```

Comment:
@shortstuffsushi this is exactly what I intend for thsi class. I'd rather move final statics like the end point, URL, version number, etc all in one place so it's never manually specified else where. 

@qinweigong in that case that will be a separate PR then.